src/DependencyInjection/Configuration.php

```diff
@@ -41,6 +41,17 @@ public function getConfigTreeBuilder()
 
         $this->addBundlesSection($root);
 
+        $root
+            ->children()
+                ->arrayNode('ivory_ckeditor')
+                    ->addDefaultsIfNotSet()
+                    ->canBeEnabled()
```

Comment:
I copied from simple-cms-bundle, there was an auto and i removed it ;-) I will redo it.